ui/src/App.css

```
@@ -36,3 +36,48 @@
     transform: rotate(360deg);
   }
 }
+
+input {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 20px;
+  margin-bottom: 20px;
```

There is a shorthand version of the `margin` property, which looks kind of like this:
```suggestion
  margin: 20px auto;
```
This bunches up top and bottom, and left and right into less lines than before. The suggestion is equivalent to what you wrote, so it's cool to straight-up add. (note the first parameter is top-bottom margin, second is left-right margin).